Source/DataSources/KmlDataSource.js

```diff
@@ -1150,7 +1150,7 @@ define([
                 entity.corridor = corridor;
                 corridor.positions = coordinates;
                 if (defined(polyline)) {
-                    corridor.material = defined(polyline.material) ? polyline.material.color : Color.WHITE;
+                    corridor.material = defined(polyline.material) ? polyline.material.color.getValue() : Color.WHITE;
```

Comment:
Yes, colors in KML are always constant (excluding the `highlight` style in `StyleMap`s, which isn't supported in Cesium). I would just change it to use `Iso860.MINIMUM_VALUE` as @mramato suggested.